app/brave_generated_resources.grd

```diff
@@ -1278,6 +1278,9 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         <message name="IDS_PLAYLIST_A11Y_PREVIOUS" desc="An a11y name for a 'prev' button in media player">
           Previous Track
         </message>
+        <message name="IDS_PLAYLIST_A11Y_TOGGLE_MUTED" desc="An a11y name for a 'mute' button in media player">
+          Toggle muted
```

Comment:
I think `Toggle mute` would be a bit more common. cc: @rmcfadden3

Reply:
@mkarolin — In the screenshot shared in this task, it appears this would be hover text for the volume/mute button…am I understanding correctly?

If yes, I would just note it's not really the toggle that's being muted, but either the track or the player volume itself. So maybe `Track muted` or `Player muted` instead? Or, if it's hover text for an action the user has not yet taken, could do `Mute track` or `Mute playlist` or something. But plz do tell me if I'm misreading.

Also, side note: Unless this is for iOS, noting the `Previous Track` thing should be sentence case. So `Previous track` instead.

Reply:
I believe it's the accessibility text for the button that toggles mute/unmute.

Reply:
Ah. @mkarolin so if that's the case, then just `Mute toggle` would be a good noun description for a11y.

Reply:
It seems I named them wrong. The strings are used for buttons title attribute, so IDR_PLAYLIST_TOOLTIP_TOGGLE_MUTED would be better name, and same for others? Then should I change the other text together?

Reply:
Hmm, yeah, I'd say if these are actually tooltips then we should rename all of the IDS_PLAYLIST_A11Y_*  to IDS_PLAYLIST_TOOLTIP_ for clarity. 

Since the tooltips here are calls for action I'd say "Toggle mute" should work if you don't want to look at the current state. Otherwise, "Mute" and "Unmute" as Ryan suggested would be better.

Reply:
Thanks, done! Could you take a look again? Also updated "Next/Previous Track" => "Next/Previous track" as @rmcfadden3 said.